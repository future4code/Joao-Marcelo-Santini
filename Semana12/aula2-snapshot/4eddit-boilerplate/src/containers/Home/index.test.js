import React from "react";
import { Home } from "./index";
import renderer from "react-test-renderer";
import {shallow} from "enzyme";

describe("Testa actions do 4eddit", () =>{
    test("Snapshot", () => {
        const tree = renderer.create(<Home goToHome={()=>{}} goToLogin={()=>{}} />)
        .toJSON();
        expect(tree).toMatchSnapshot();
    });

});