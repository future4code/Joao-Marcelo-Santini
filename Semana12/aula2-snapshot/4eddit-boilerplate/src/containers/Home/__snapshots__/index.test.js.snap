// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testa actions do 4eddit Snapshot 1`] = `
<div
  className="sc-fzXfMB gdDxBP"
>
  <div
    className="sc-fzXfMC dCnWJT"
  >
    <img
      className="sc-fzXfME dmgdzZ"
      src="logo.png"
    />
    <div
      className="sc-fzXfMD hILBuc"
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSizeLarge MuiButton-sizeLarge"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Cadastrar
        </span>
      </button>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSizeLarge MuiButton-sizeLarge"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Login
        </span>
      </button>
    </div>
  </div>
</div>
`;
